WTWSMS/common/landed_titles/mercenaries.txt

```diff
@@ -2205,6 +2196,7 @@ d_mamluks = {
 	# Hire Trigger
 	allow = {
 		religion_group = muslim
+		year = 800
```

Comment:
I added the year trigger because the Mamluks did not exist until after the Sunni/Shia split. The Mamluk slave brigades, as we know them, did not exist right at the dawn of Islam.